### PR TITLE
Configure GoPhish to use TLS

### PR DESCRIPTION
**Pull request by jsf9k**
## 🗣 Description ##

This pull request:
- Removes some quote characters that are no longer necessary (or correct) after cisagov/pca-gophish-composition#38.
- Changes the GoPhish published port from 80 to 443.
- Configures GoPhish to use TLS.

## 💭 Motivation and context ##

See cisagov/cool-system-internal#81.

See also:
- cisagov/pca-gophish-composition#38
- cisagov/cool-assessment-terraform#166

## 🧪 Testing ##

All automated tests pass.  I also built a new staging AMI with these changes, deployed it to `env6-staging`, and verified that it functions as expected.

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - _eschew scope creep!_
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All new and existing tests pass.

## ✅ Pre-merge checklist ##

- [x] Revert to the default branch of [cisagov/ansible-role-pca-gophish-composition](https://github.com/cisagov/ansible-role-pca-gophish-composition).
- [x] Finalize version.

## ✅ Post-merge checklist ##

- [x] Create a release.

### CODE REVIEW

File: src/pca_gophish.yml [hunk 1]
```diff
@@ -52,12 +52,18 @@
...
             system: yes
-        - name: Change Gophish published port from 3380 to 80
+        - name: Change Gophish published port from 3380 to 443
           ansible.builtin.lineinfile:
             path: "{{ pca_gophish_composition_path }}/docker-compose.yml"
             regexp: "published: 3380$"
-            line: "        published: 80"
+            line: "        published: 443"
             state: present
+        - name: Configure gophish to use TLS
```
Comment (dav3r) [inline src/pca_gophish.yml]: Capitalization nitpick:
```suggestion
        - name: Configure Gophish to use TLS
```

File: src/pca_gophish.yml [hunk 2]
```diff
@@ -52,12 +52,18 @@
...
             regexp: "published: 3380$"
-            line: "        published: 80"
+            line: "        published: 443"
             state: present
+        - name: Configure Gophish to use TLS
+          ansible.builtin.replace:
+            path: >-
+              {{ pca_gophish_composition_path }}/secrets/gophish/config.json
+            regexp: '^(\s*"use_tls":\s*)false(\s*)$'
+            replace: '\1true'
```
Comment (mcdonnnj) [inline src/pca_gophish.yml:66]: Based on the regexp do you need to add the other matching group?

```suggestion
            replace: '\1true\2'
```

**Review by mcdonnnj — APPROVED** [review]
Makes sense. I did have one question for you.

File: src/pca_gophish.yml (hunk 2 above)
Comment (jsf9k) [reply src/pca_gophish.yml:66]: I don't think so, since it's just pointless whitespace.  Second matching group removed in commit 6a23b30.

**Review by dav3r — APPROVED** [review]
Outstanding! 💎

**Pull request merged**